application/forms/PreferenceForm.php

```diff
@@ -202,6 +203,33 @@ public function createElements(array $formData)
             }
         }
 
+        if (! isset($formData['theme'])
+            || $formData['theme'] === StyleSheet::DEFAULT_THEME
+            || strpos(file_get_contents(StyleSheet::getThemeFile($formData['theme'])), 'prefers-color-scheme') !== false
+            || strpos(file_get_contents(StyleSheet::getThemeFile($formData['theme'])), 'min-height') === false
```

Comment:
You can't just search for `min-height`, this is a standard CSS property, that can occur anywhere in the file.